tests/test_framework/phpbb_ui_test_case.php

```diff
@@ -458,7 +459,7 @@ protected function get_extension_manager()
 			$container,
 			$db,
 			$config,
-			$user,
```

Comment:
This argument is incorrect, but it wasn't giving any errors, maybe this method is never executed